components/brave_rewards/common/rewards_util.cc

```diff
@@ -40,10 +45,49 @@ bool IsDisabledByFeature() {
   return false;
 }
 
+bool IsOFACSanctionedRegion(const std::string& country_code) {
+  return brave_l10n::IsISOCountryCodeOFACSanctioned(country_code) ||
+         brave_l10n::IsUNM49CodeOFACSanctioned(country_code);
```

Comment:
We should discuss whether we should also call `brave_l10n::IsUNM49CodeOFACSanctioned` because, for example, some OS regional settings can return `en_001` for the locale meaning `the world`, and this would block all countries. cc @Miyayes @jsecretan

Reply:
@tmancey wouldn't it return `false` if we pass `en_001`? I assumed only countries on the sanctioned list would return `true`.

Reply:
UN M.49 codes encapsulate larger areas, i.e. `en_001` would encapsulate the entire World. UN M.49 codes are returned based on specific configurations so are not as common as `en_US` etc.

Reply:
For ads we will only check `brave_l10n::IsISOCountryCodeOFACSanctioned(country_code)` as we will also have server side mitigation. For Rewards I think we should discuss in a DM before also using `brave_l10n::IsUNM49CodeOFACSanctioned(country_code)`

Reply:
As per DM discussion removed to call to `brave_l10n::IsUNM49CodeOFACSanctioned`